zcash_client_sqlite/src/wallet/init/migrations/addresses_table.rs

```diff
@@ -52,6 +52,8 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                 account INTEGER NOT NULL,
                 diversifier_index_be BLOB NOT NULL,
                 address TEXT NOT NULL,
+                sapling_receiver TEXT,
+                transparent_receiver TEXT,
```

Comment:
This is only for caching purposes, and should not be relied on externally. Name the column to indicate this:
```suggestion
                cached_transparent_receiver TEXT,
```